<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Текстовый флаг</string>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>550</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Верхняя полоса</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QRadioButton" name="top_red">
         <property name="text">
          <string>Красный</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="top_blue">
         <property name="text">
          <string>Синий</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="top_green">
         <property name="text">
          <string>Зеленый</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="top_yellow">
         <property name="text">
          <string>Желтый</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="top_white">
         <property name="text">
          <string>Белый</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Средняя полоса</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QRadioButton" name="middle_red">
         <property name="text">
          <string>Красный</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="middle_blue">
         <property name="text">
          <string>Синий</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="middle_green">
         <property name="text">
          <string>Зеленый</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="middle_yellow">
         <property name="text">
          <string>Желтый</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="middle_white">
         <property name="text">
          <string>Белый</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Нижняя полоса</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QRadioButton" name="bottom_red">
         <property name="text">
          <string>Красный</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="bottom_blue">
         <property name="text">
          <string>Синий</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="bottom_green">
         <property name="text">
          <string>Зеленый</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="bottom_yellow">
         <property name="text">
          <string>Желтый</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="bottom_white">
         <property name="text">
          <string>Белый</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="drawButton">
      <property name="text">
       <string>Нарисовать</string>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px; font-weight: bold;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultLabel">
      <property name="text">
       <string>Выберите цвета полос</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 16px; font-weight: bold; padding: 15px; background-color: #f0f0f0; border-radius: 5px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>